src/components/shared/field.svelte

```diff
@@ -0,0 +1,119 @@
+<script>
+import BSN from "bootstrap.native/dist/bootstrap-native.esm.min.js";
+import { createEventDispatcher, onMount } from "svelte";
+
+const dispatch = createEventDispatcher();
+
+// Props
+export let item = {};
+export let value = null;
+
+// Auxiliary variables
+let dropdown = null;
+
+// Bindings
+let dropdownButton;
+let popoverIcon;
+
+// Computed variables
+const isCheckbox = item.type === "checkbox";
+const isInput = item.type === "input";
+const isDropdown = item.type === "dropdown";
+
+// Methods
+function onChange(newValue) {
+  if (newValue !== value) {
+    dispatch("change", newValue);
+  }
+
+  dropdown.toggle();
+}
+
+// Hooks
+onMount(() => {
+  if (isDropdown) {
+    // Initialize dropdown
+    dropdown = new BSN.Dropdown(dropdownButton);
+
+    // Initialize popover
+    const providers =
+      item.key === "mergeSearchProviders" ? "search providers" : "queries";
+    new BSN.Tooltip(popoverIcon, {
+      customClass: "ml-1",
+      title: `<div class='text-left'><div><strong>Merge:</strong> Adds ${providers} that aren't already in current list of ${providers}.</div><div><strong>Override:</strong> Replaces local ${providers} with new settings.</div><div><strong>Ignore:</strong> Keeps current list and ignores any incoming changes.</div></div>`,
```

Comment:
One of the reasons to switch to Svelte was to avoid these kind of ugly code.

According to [BSN docs](https://thednp.github.io/bootstrap.native/#componentTooltip), you can pass a DOM node as tooltip, so _I think_ you could do something like:

```js
new BSN.Tooltip(popoverIcon, {
      customClass: "ml-1",
      title: popoverTitle
```

```html
    <i
      bind:this="{popoverIcon}"
      class="fas fa-info-circle text-info"
      aria-hidden="true"
      data-toggle="tooltip"
      data-placement="right"></i>
    <div bind:this="{popoverTitle}" class='text-left'><div><strong>Merge:</strong>...</div></div>
```